app/src/main/java/yuricfurusho/traderev/PhotoAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package yuricfurusho.traderev
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import kotlinx.android.synthetic.main.adapter_photos_item.view.*
+import yuricfurusho.traderev.photos.Photo
+
+class PhotoAdapter : RecyclerView.Adapter<PhotoAdapter.PhotosViewHolder>() {
```

Comment:
I saw that I could use ListAdapter, but decided to stick with the pattern I was used to for now.